tmpl/volk_cpu.tmpl.c

```diff
@@ -73,7 +73,7 @@ struct VOLK_CPU volk_cpu;
 
 static inline unsigned int cpuid_count_x86_bit(unsigned int level, unsigned int count, unsigned int reg, unsigned int bit) {
 #if defined(VOLK_CPU_x86)
-    unsigned int regs[4] = {0};
+    int regs[4] = {0};
```

Comment:
the rest of the code wants this as an unsigned int, except for the signature of the windows call. I'd much rather see a cast inside the MSVC specific code rather than blindly changing types to satisfy MSVC warnings